server/client_server.go

```diff
@@ -22,10 +30,120 @@ func newServerClient(jiraClient *jira.Client) Client {
 	}
 }
 
+type IssueInfo struct {
```

Comment:
Maybe rename this to `ProjectIssueInfo` to note that its info about a specific project